lib/companies/schema/company.ex

```diff
@@ -1,27 +1,33 @@
 defmodule Companies.Schema.Company do
   @moduledoc false
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
   alias Companies.Industries
 
-  # @enforce_keys [:industry, :last_changed_on, :location, :name, :website]
+  schema "companies" do
+    field :blog, :string
+    field :description, :string
+    field :github, :string
```

Comment:
We should validate the format to a URL with `github.com` as the host, and no `/` in the path.

Reply:
For future, we could make this more generic of `field :open_source_link, :string` to be inclusive of gitlab or bitbucket.